Common/Tools/BuildTasks/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("Microsoft.VisualStudioTools.BuildTasks.Properties")]
```

Comment:
Remove `.Properties` from title and product

Reply:
Fixed in next commit